app/helpers/content_helper.rb

```diff
@@ -6,4 +6,8 @@ def path_for_this_page(page)
   def is_current?(page)
     page.slug == params[:slug]
   end
+
+  def find_page_with_title(title)
+    ContentPage.find_by_title(title)
```

Comment:
I feel like this belongs in the controller. 

Assigning this to a variable and then using it in the view feels like a cleaner way to make a database call to fetch content rather than using a view helper.

Reply:
Agree with tony 100% this needs to be an instance variable in the corresponding controller action. Database queries should not take place in any part of the views.

Reply:
Ok will do, though the mobile menu layout view partial makes lots of database calls too.

Reply:
The longer term answer is to introduce a featured page attribute, 'Get help with your practice' would have an attribute set, featured_page: 1,  and then the view would just render that featured page.  Rather than hard coding the page title in the view.

Reply:
> Ok will do, though the mobile menu layout view partial makes lots of database calls too.

good spot, we need to refactor that at some point

Reply:
![Screenshot 2021-03-08 at 15 00 06](https://user-images.githubusercontent.com/3352035/110338798-539be800-801f-11eb-8d6d-eef819a5d9be.png)

Here you go